dojo/api_v2/serializers.py

```diff
@@ -1280,13 +1279,15 @@ def save(self, push_to_jira=False):
         _, test_title, scan_type, engagement_id, engagement_name, product_name, product_type_name, auto_create_context = get_import_meta_data_from_dict(data)
         engagement = get_or_create_engagement(engagement_id, engagement_name, product_name, product_type_name, auto_create_context)
 
+        # have to make the scan_date_time timezon aware otherwise uploads via the API would fail (but unit tests for api upload would pass...)
```

Comment:
Typo in timezone (missing e)

Reply:
actually that word triggers the python runtime to go into overdrive mode

Reply:
typo fixed.